project-metrics/metrics_service/server.py

```diff
@@ -61,6 +61,36 @@ def list_metrics():
                        }), status.HTTP_200_OK
 
 
+@app.route('/api/badge/<metric_cls_name>')
+def metric_badge(metric_cls_name):
+  """Provides a response for sheilds.io to render a badge for GitHub.
+
+  See https://shields.io/endpoint.
+  """
+  response = {
+      'schemaVersion': 1,
+      'color': 'lightgray',
+      'label': 'Unknown',
+      'message': '?',
+  }
+  try:
+    metric = base.Metric.get_latest()[metric_cls_name]
+    response['color'] = [
+        '#EEEEEE',
+        'indianred',
+        'orange',
+        'greenyellow',
+        'lawngreen',
+        'forestgreen',
+    ][metric.score.value]
```

Comment:
Put the colour values in a const at the top of the files and reference it here

Reply:
Done.